Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -89,6 +89,17 @@ int RGroupDecomposition::add(const ROMol &inmol) {
   const bool addCoords = true;
   MolOps::addHs(mol, explicitOnly, addCoords);
 
+  // mark any wildcards in input molecule:
+  for (auto &atom : mol.atoms()) {
+    if (atom->getAtomicNum() == 0) {
+      atom->setIsotope(1000U);
```

Comment:
Might be better to use a tag here.  I for one have used 1000 isotopes as a tag in the past...

Reply:
something like
```
atom->setBoolProp("WASDUMMY", true);
```